pages/CartPage.ts

```diff
@@ -0,0 +1,32 @@
+import { Page, Locator, expect } from '@playwright/test';
+
+export class CartPage {
+  readonly page: Page;
+  readonly productName: Locator;
+  readonly unitPrice: Locator;
+  readonly quantity: Locator;
+  readonly totalPrice: Locator;
+  readonly proceedToCheckoutButton: Locator;
+
+  constructor(page: Page) {
+    this.page = page;
+    this.productName = page.locator('[data-test="product-title"]');
```

Comment:
You can make these changes https://playwright.dev/docs/api/class-testoptions#test-options-test-id-attribute, and then you can use getByTestId() method - https://playwright.dev/docs/api/class-locator#locator-get-by-test-id. 

```suggestion
    this.productName = page.getByTestId('product-title');
```

More clear way